packages/renderer/src/lib/recommendation/ExtensionBanner.spec.ts

```
@@ -160,10 +160,10 @@ test('opening messageBox and hiding banner', async () => {
   closeButton.click();
 
   expect(window.showMessageBox).toBeCalledWith({
-    title: 'Hide extension recommendation banners',
+    title: 'Hide Extension Recommendations?',
     message: `Do you want to hide extension recommendation banners?`,
```

Biome (2.4.11)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)